tests/e2e/service-worker.js

```diff
@@ -75,8 +75,7 @@ describe('Service worker cache', () => {
       '/img/icon-pregnant.svg',
       '/img/setup-wizard-demo.png',
       '/img/simprints.png',
-      '/js/inbox.js',
-      '/js/templates.js',
```

Comment:
Shouldn't `index.html` also be here somewhere?

Reply:
I did a research and testing. I learned that none of the index.html files (login, webapp..) are here because they are  mapped in the dynamic url "cache" (`dynamicUrlToDependencies`) and not contained in the `sw-precache`. 
The `sw-precache` contains static resources like the `audio.mp3`. Actually I haven't found where the dynamic url is saved or examples of how to e2e test it, then I couldn't add a test targeting this :(

I manually tested in my laptop and phone with the internet off and the app looks like it's working fine, I was able to create a report and a contact.

Reply:
> Actually I haven't found where the dynamic url is saved or examples of how to e2e test it, then I couldn't add a test targeting this :(

I think we're already testing this: 
https://github.com/medic/cht-core/blob/master/tests/mobile/service-worker.specs.js#L98

Reply:
I see! Thanks for sharing, I've added a couple of checks for the login's html and the appcache-upgrade's html